contracts/multisig-prover/src/state.rs

```diff
@@ -12,6 +12,7 @@ use crate::types::{BatchId, CommandBatch};
 #[cw_serde]
 pub struct Config {
     pub admin: Addr,
+    pub governance: Addr,
```

Comment:
Now that we have the contracts deployed already on the testnet, doesn't this require some migration?

Reply:
yeah if we want to upgrade, as oppose to redeploy. The prover can pretty easily just be redeployed.